scripts/verify.py

```diff
@@ -37,12 +37,17 @@ def main(checks):
             run("./scripts/check_docs.py", shell=True, check=True)
             print("check docs passed")
 
+        if "check-links" in checks:
+            print("LinksValidity (check):", flush=True)
```

Comment:
"Checking links in Markdown files" might be a better comment.

Reply:
Cool. Actually, I was trying to match the style of other items like `Documentation (build):`, `Linter (pylint):` .. etc etc.